<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMROEditor_T0</class>
 <widget class="QDialog" name="IMROEditor_T0">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>556</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Imec Readout Table Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>18</number>
   </property>
   <property name="verticalSpacing">
    <number>15</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="prbHdr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Probe type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="prbLbl">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>71</red>
             <green>3</green>
             <blue>34</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>71</red>
             <green>3</green>
             <blue>34</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>106</red>
             <green>104</green>
             <blue>100</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <row>
      <property name="text">
       <string>0</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Bank</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Refid</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>APgain</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LFgain</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>APfilt</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="16" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>APfilt</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="apBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set All</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="apCB">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>125</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>APgain</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Row</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>14</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>14</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="rowSB">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="bankSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Refid</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>14</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="lfBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set All</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QComboBox" name="hipassCB">
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>300Hz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="refidCB">
       <property name="maxVisibleItems">
        <number>14</number>
       </property>
       <item>
        <property name="text">
         <string>0 = Ext</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 = Tip</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 = 192</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 = 576</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 = 960</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QComboBox" name="lfCB">
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>125</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>LFgain</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="bankBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set All</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="refidBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set All</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QPushButton" name="hipassBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set All</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>14</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Bank</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>14</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="blockBut">
       <property name="text">
        <string>Center Here</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="statusLbl">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>3</green>
           <blue>34</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>71</red>
           <green>3</green>
           <blue>34</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>18</number>
     </property>
     <property name="rightMargin">
      <number>18</number>
     </property>
     <item>
      <widget class="QPushButton" name="loadBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpBut">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>defaultBut</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>loadBut</tabstop>
  <tabstop>saveBut</tabstop>
  <tabstop>helpBut</tabstop>
  <tabstop>rowSB</tabstop>
  <tabstop>blockBut</tabstop>
  <tabstop>bankSB</tabstop>
  <tabstop>bankBut</tabstop>
  <tabstop>refidCB</tabstop>
  <tabstop>refidBut</tabstop>
  <tabstop>apCB</tabstop>
  <tabstop>apBut</tabstop>
  <tabstop>lfCB</tabstop>
  <tabstop>lfBut</tabstop>
  <tabstop>hipassCB</tabstop>
  <tabstop>hipassBut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
